docs/react/sponsor.en-US.md

```
@@ -15,7 +15,11 @@ We'd like to thank the following companies and individuals for their generous sp
 
 ## Sponsors [![](https://opencollective.com/ant-design/tiers/sponsors/badge.svg?label=Sponsors&color=brightgreen)](https://opencollective.com/ant-design/contribute/sponsors-218)
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)